src/css/app.sass

```diff
@@ -13,6 +13,14 @@ body
     border-radius: 4px
     color: white
 
+.q-card
+    border-radius: 20px
+
+.q-card 
+    .q-img
```

Comment:
Rounding should be on card not img. otherwise it does this weird thingy

![image](https://user-images.githubusercontent.com/14140515/176620411-4c4824cf-a187-41d9-9a6e-51a6d7891559.png)